Makefile

```diff
@@ -22,39 +22,39 @@ release:
 
 	GOOS=darwin GOARCH=amd64 go build -ldflags "-X main.versionNumber=$(TRAVIS_TAG)" -o release/terraform-provider-alks_v$(TRAVIS_TAG) -mod=vendor $(package)
 	chmod +x release/terraform-provider-alks_v$(TRAVIS_TAG)
-	tar -cvzf release/terraform-provider-alks-darwin-amd64.tar.gz -C release/ terraform-provider-alks_v$(TRAVIS_TAG)
+	tar -cvzf release/terraform-provider-alks_$(TRAVIS_TAG)_darwin_amd64.tar.gz -C release/ terraform-provider-alks_v$(TRAVIS_TAG)
```

Comment:
I saw your note about changing the file name format.  One other thing I was curious about though, is in the example you shared, every archive was a zip file rather than the split we have here between tar files and zips.  Does HashiCorp specify the format?

Reply:
I emailed the HashiCorp employee about this very thing, and still have not received a reply yet. But you bring up a good point... let me wait to see what they reply, and if it requires changes I can update this PR with the appropriate `.zip` extensions.